src/gtk/animate.cpp

```diff
@@ -12,9 +12,10 @@
 #include "wx/wxprec.h"
 
 #if wxUSE_ANIMATIONCTRL && !defined(__WXUNIVERSAL__)
```

Comment:
```suggestion
#if wxUSE_ANIMATIONCTRL
```
I don't think we need this check here any longer, as it's done in the header.